decidim-admin/spec/features/manage_participatory_processes_spec.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Manage participatory processes", type: :feature do
```

Comment:
Add an example that verifies an admin can only manage processes from its organization.